### DISCUSSION TITLE
Panel - TCP persistent mode

### DISCUSSION THREAD

**Discussion by nathanh2011** [dannagle/PacketSender#306] [Q&A] [answered]
Hi,

Is it possible to have a panel that runs more than one packet but in the same TCP connection? At the moment it seems to send all of them as separate connections. 

I am currently trying to test sending 3 packets in quick succession (1 second apart) and this would really help me.

Any help is greatly appreciated, thanks.

**Comment by dannagle** [comment]
You are correct. It makes a separate connection with each packet. It will send then disconnect. 

Allowing packets going to same destination to be a single connection is an interesting idea. The technical work to achieve that is not too bad. If you will make a [new issue](https://github.com/dannagle/PacketSender/issues/new?assignees=&labels=&template=feature_request.md&title=%5BIDEA%5D), I will flag it as a feature request.

**Comment by nathanh2011** [comment, marked as answer]
Thanks for your response. As requested, a new issue has been created. https://github.com/dannagle/PacketSender/issues/307
